src/EventGrid/EventGrid/ChangeLog.md

```diff
@@ -19,6 +19,21 @@
 -->
 ## Upcoming Release
 
+## Version 1.1.1
+* Updated to use the 2019-02-01-preview API version.
+* New commands:
+    - New-AzureRmEventGridDomainTopic:
+        - Creates a new Azure Event Grid Domain Topic.
+    - Remove-AzureRmEventGridDomainTopic:
+        - Remove an existing Azure Event Grid Domain Topic.
+
+* Update the following cmdlets to support new scenarios in 2019-02-01-preview API version
+    - New-AzureRmEventGridSubscription/Update-AzureRmEventGridSubscription:
+        - Add new enum for servicebusqueue as destination.
+        - Disllow usage of "All" in IncludesEventTypes option and replace it with $null
```

Comment:
Nit: IncludesEventTypes  should be -IncludedEventType (without the plural) per https://docs.microsoft.com/en-us/powershell/module/azurerm.eventgrid/new-azurermeventgridsubscription?view=azurermps-6.13.0.

Reply:
Fix. thx